### ISSUE TITLE
Loop_Apply not found/not installing

### ISSUE THREAD

**Issue by aadler** [hadley/plyr#266]
I installed `plyr` into a build of R-devel_2015-10-09 using the Windows (7) GCC 4.9.3 64bit toolchain in Rtools and got the same issue as https://github.com/hadley/plyr/issues/258, ""loop_apply" not resolved from current namespace". I found to suggestion on [Stack Overflow](http://stackoverflow.com/questions/31352789/keep-hitting-the-error-loop-apply-not-resolved-from-current-namespace-plyr) install from source, but I did that.

I tried installing devtools and then building plyr directly from github, and I got the error below. Installing from binary, however, seems to work fine.

~~~~
* installing *source* package 'plyr' ...
** libs
F:/R/Rtools/mingw_64/bin/g++  -I"F:/R/RCurrent/R-devel/include" -DNDEBUG  -march=native -pipe  -I"F:/R/RCurrent/R-devel/library/Rcpp/include" -I"F:/R/RLocalSoft/include"     -O2 -std=gnu++11 -c RcppExports.cpp -o RcppExports.o
F:/R/Rtools/mingw_64/bin/gcc  -I"F:/R/RCurrent/R-devel/include" -DNDEBUG  -march=native -pipe  -I"F:/R/RCurrent/R-devel/library/Rcpp/include" -I"F:/R/RLocalSoft/include"     -O3 -std=gnu99 -c loop_apply.c -o loop_apply.o
F:/R/Rtools/mingw_64/bin/g++  -I"F:/R/RCurrent/R-devel/include" -DNDEBUG  -march=native -pipe  -I"F:/R/RCurrent/R-devel/library/Rcpp/include" -I"F:/R/RLocalSoft/include"     -O2 -std=gnu++11 -c split-numeric.cpp -o split-numeric.o
F:/R/Rtools/mingw_64/bin/g++ -shared -s -static-libgcc -o plyr.dll tmp.def RcppExports.o loop_apply.o split-numeric.o -LF:/R/RLocalSoft/lib/x64 -LF:/R/RLocalSoft/lib -LF:/R/RCurrent/R-devel/bin/x64 -lR
installing to F:/R/RCurrent/R-devel/library/plyr/libs/x64
** R

[snip]

** testing if installed package can be loaded
Error in FUN(X[[i]], ...) : 
  no such symbol loop_apply_ in package F:/R/RCurrent/R-devel/library/plyr/libs/x64/plyr.dll
Error: loading failed
Execution halted
ERROR: loading failed
* removing 'F:/R/RCurrent/R-devel/library/plyr'
~~~~
Thank you.

**Comment by aadler** [comment]
For what it is worth, my current workaround is to remove -std=c++11 (or gnu++11) from CXXFLAGS before installing plyr (and [dplyr](https://github.com/hadley/dplyr) for that matter). Then it installs and runs without a problem

**Comment by aadler** [comment]
I think this was fixed by 3256bb524dbe04982ad54300057dd1e1bf852906
